Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -106,7 +106,7 @@ public void ProcRunner_FailsOnTimeout()
             // Assert
             timer.Stop(); // Sanity check that the process actually timed out
             logger.LogInfo("Test output: test ran for {0}ms", timer.ElapsedMilliseconds);
-            // TODO: the following line throws regularly on the CI machines (elapsed time is around 97ms);
+            // TODO: the following line throws regularly on the CI machines (elapsed time is around 97ms)
             // Assert.IsTrue(timer.ElapsedMilliseconds >= 100, "Test error: batch process exited too early. Elapsed time(ms): {0}", timer.ElapsedMilliseconds);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS125)